docs/our_work/a_and_e_forecasting_tool.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Accident and Emergency (A&E) Forecasting Tool'
+summary: 'A probabilistic model which gives a three-week forecast for A&E departments, predicting their expected admissions loads'
+origin: ''
+tags: ['MODELLING', 'HOSPITAL', 'MONTE CARLO', 'MCMC', 'URGENT CARE', 'ACCIDENT AND EMERGENCY']
+---
+
```

Comment:
do you have an image that could serve as a visual abstract for the project? COuld you make one using AI? This means it can get used for stuff like the carousel on the home page!

Reply:
I've added an image at `images/a_and_e_forecasting/overview.png`. I don't know how to include this on the carousel - could you handle that end?